src/components/Card.js

```diff
@@ -1,49 +1,54 @@
-import { useContext } from 'react';
-import { AppContext } from '../AppContext';
-
-const Card = (card) => {
-  const { nickName } = card;
-  const { inputCard } = useContext(AppContext);
+const Card = (props) => {
+  const { type = 'small', card } = props;
```

Comment:
```suggestion
const Card = ({ type = 'small', card }) => {
```
파라미터를 구조분해 할당으로 바로 작성하시는게 어떨까요?

Reply:
예 불필요한 로직이네요 제안주신대로 수정하겠습니다